forte/data/readers/base_reader.py

```diff
@@ -243,7 +242,6 @@ def cache_data(self,
         :func:`serialize_instance`).
 
         Args:
-            cache_directory: The directory used to cache the data.
             collection: The collection to be read as a DataPack, this collection
```

Comment:
The collection is a piece of data from the ``_collect`` function,  to be read to produce DataPack(s). During caching, a cache key is computed based on the data in this collection.

Reply:
Will update